### ISSUE TITLE
pandoc-2.11.3 test suite failure

### ISSUE THREAD

**Issue by peti** [jgm/pandoc#6961]
Citing from https://hydra.nixos.org/build/133145578/nixlog/1:

~~~
    jats
      writer
        jats_archiving
          basic:                                                     Could not find data file /nix/store/pl5q6l1kfal7z4r1h85g3s2mlxp4649g-pandoc-2.11.3-data/share/ghc-8.10.2/x86_64-linux-ghc-8.10.2/pandoc-2.11.3/data/templates/affiliations.jats
FAIL (0.06s)
            user error (Pandoc failed with error code ExitFailure 97)
          tables:                                                    OK (0.11s)
          planets table:                                             OK (0.09s)
            Golden file did not exist; created
          nordics table:                                             OK (0.06s)
            Golden file did not exist; created
          students table:                                            OK (0.06s)
            Golden file did not exist; created
        jats_articleauthoring
          basic:                                                     Could not find data file /nix/store/pl5q6l1kfal7z4r1h85g3s2mlxp4649g-pandoc-2.11.3-data/share/ghc-8.10.2/x86_64-linux-ghc-8.10.2/pandoc-2.11.3/data/templates/affiliations.jats
FAIL (0.05s)
            user error (Pandoc failed with error code ExitFailure 97)
          tables:                                                    OK (0.11s)
        jats_publishing
          basic:                                                     Could not find data file /nix/store/pl5q6l1kfal7z4r1h85g3s2mlxp4649g-pandoc-2.11.3-data/share/ghc-8.10.2/x86_64-linux-ghc-8.10.2/pandoc-2.11.3/data/templates/affiliations.jats
FAIL (0.06s)
            user error (Pandoc failed with error code ExitFailure 97)
          tables:                                                    OK (0.12s)
      reader:                                                        OK (0.20s)
~~~

It seems like pandoc expects those files in their respective installation locations even though the package hasn't been installed yet; the test suite is run after the build phase but before the install phase.

**Comment by jgm** [comment]
They weren't included in pandoc.cabal; that's the problem.
I thought we had something in CI to catch these cases, but I guess that needs to be revisited.

**Comment by tarleb** [comment]
Fixed in pandoc 2.11.3.1.

**Comment by jgm** [comment]
I've added CI checks that ensure that the cabal file's data and test files match what is in the repository.
